src/main/java/org/jabref/model/util/FileHelper.java

```diff
@@ -115,7 +116,9 @@
         Objects.requireNonNull(directory);
 
         Path file = Paths.get(filename);
-        if (Files.exists(file)) {
+        //Explicitly check for an empty String, as File.exists returns true on that empty path, because it maps to the default jar location
```

Comment:
Will do that.
The failing tests are still somehow weird as locally the run through partly.. I still think there's something with the prefs which has some dubious side effects